homeassistant/components/apache_kafka/__init__.py

```diff
@@ -32,6 +36,9 @@
                 vol.Required(CONF_PORT): cv.port,
                 vol.Required(CONF_TOPIC): cv.string,
                 vol.Optional(CONF_FILTER, default={}): FILTER_SCHEMA,
+                vol.Optional(CONF_SECURITY_PROTOCOL, default="PLAINTEXT"): cv.string,
+                vol.Optional(CONF_USERNAME, default=""): cv.string,
+                vol.Optional(CONF_PASSWORD, default=""): cv.string,
```

Comment:
Don't need a default if it's going to be falsy – `vol.Optional` will default to `None`.